example/android/app/src/main/res/values/dark_theme.xml

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ dark_theme.xml
+  ~
+  ~   PSPDFKit
+  ~
+  ~   Copyright © 2018-2019 PSPDFKit GmbH. All rights reserved.
+  ~
+  ~   THIS SOURCE CODE AND ANY ACCOMPANYING DOCUMENTATION ARE PROTECTED BY INTERNATIONAL COPYRIGHT LAW
+  ~   AND MAY NOT BE RESOLD OR REDISTRIBUTED. USAGE IS BOUND TO THE PSPDFKIT LICENSE AGREEMENT.
+  ~   UNAUTHORIZED REPRODUCTION OR DISTRIBUTION IS SUBJECT TO CIVIL AND CRIMINAL PENALTIES.
+  ~   This notice may not be removed from this file.
+  -->
+
+<resources>
+
+    <style name="PSPDFCatalog.Theme.Dark" parent="PSPDFKit.Theme.Dark">
```

Comment:
Same style name change suggested here, and for all other styles that are defined too.